nuto/math/CubicSplineInterpolation.cpp

```diff
@@ -42,7 +42,7 @@ double NuTo::Math::CubicSplineInterpolation::operator()(double x)
     return y;
 }
 
-double NuTo::Math::CubicSplineInterpolation::derivative(double x)
+double NuTo::Math::CubicSplineInterpolation::derivative(double x) const
```

Comment:
I know, you did not touch that, but for our naming convention, this could be changed to a capital `D` in `Derivative`, same goes for `Bisection`.